ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -2969,6 +2969,34 @@ def test_numpy_instance_setitem(
     )
 
 
+#__setstate__
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="__setstate__",
+    state={"ivy_array": np.array},
```

Comment:
you should use helper functions to generate inputs for arguments, not listing it out directly like this. you can refer to other tests in the file for examples.
_ test_numpy_instance_setstate[cpu-ivy.functional.backends.tensorflow-False-False] _
ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:2975: in wrapped_test
    init_tree="numpy.array",
E   hypothesis.errors.InvalidArgument: test_numpy_instance_setstate() got an unexpected keyword argument 'state', from `state={'ivy_array': <built-in function array>}` in @given

Reply:
Ok got it, sir, 
Thanks for the review.

Reply:
@sladyn98

Reply:
I just found out that maybe your implementation of _setstate_ may not be correct as it is different form the definition of numpy version(https://numpy.org/doc/stable/reference/generated/numpy.ndarray.__setstate__.html)

Reply:
sorry for the delay response, I'm working on it